.circleci/config.yml

```diff
@@ -83,7 +83,7 @@ commands:
         type: string
         default: "/var/www/code/drush/sites/self.site.yml"
     steps:
-      - run: "curl -f -o << parameters.destination >> -L https://$GITHUB_DS_INFRASTRUCTURE_TOKEN@raw.githubusercontent.com/massgov/DS-Infrastructure/develop/docs/massgov/self.site.yml"
+      - run: "curl -f -o << parameters.destination >> -L https://$GITHUB_MASSGOV_SELF_SITE_YML_TOKEN@raw.githubusercontent.com/massgov/massgov-internal-docs/master/self.site.yml"
```

Comment:
@a-username @rbayliss 
After token refactoring, I am going to checkout CircleCI's `config.yml` into this branch.